fe/src/main/java/org/apache/doris/load/BrokerFileGroup.java

```diff
@@ -228,6 +231,8 @@ public void write(DataOutput out) throws IOException {
         Text.writeString(out, valueSeparator);
         // lineDelimiter
         Text.writeString(out, lineDelimiter);
+        // format type
+        Text.writeString(out, fileFormat);
```

Comment:
you can't write this value without change META_VERSION. this can make frontend unable to load old image.

Now, we don't persist this value until we have enough meta change.

Reply:
ok